pkg/operation/seed/seed.go

```diff
@@ -761,6 +778,24 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 		return err
 	}
 
+	// managed nginx-ingress handling
+	if err := handleIngressDNSEntry(ctx, k8sSeedClient, chartApplier, seed); err != nil {
+		return err
+	}
+
+	var ingressClass string
+	if managedIngress(seed) {
+		ingressClass = v1beta1constants.SeedNginxIngressClass
+	} else {
+		ingressClass = v1beta1constants.ShootNginxIngressClass
```

Comment:
Yes, we have, see https://github.com/gardener/gardener/blob/master/docs/deployment/deploy_gardenlet_manually.md#example-nginx-ingress-controller (the document should have been more clear that it has to be nginx because this is the hard-coded ingress class in gardener).

Reply:
ok thanks